net-libs/mbedtls/mbedtls-2.25.0.recipe

```diff
@@ -0,0 +1,151 @@
+SUMMARY="Open source, portable, easy to use, readable and flexible SSL library"
+DESCRIPTION="mbed TLS (formerly known as PolarSSL) makes it trivially easy for \
+developers to include cryptographic and SSL/TLS capabilities in their (embedded) \
+products, facilitating this functionality with a minimal coding footprint."
+HOMEPAGE="https://tls.mbed.org/"
+COPYRIGHT="2006-2018 ARM Limited"
+LICENSE="Apache v2"
+REVISION="1"
+SOURCE_URI="https://github.com/ARMmbed/mbedtls/archive/mbedtls-$portVersion.tar.gz"
+CHECKSUM_SHA256="ea2049c2dd4868693998d5a9780e198194be5aea1706ff4a9d4f882f18c0a101"
+SOURCE_DIR="mbedtls-mbedtls-$portVersion"
+PATCHES="mbedtls-$portVersion.patchset"
+
+ARCHITECTURES="!x86_gcc2 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+libVersion="$portVersion"
+
+PROVIDES="
+	mbedtls$secondaryArchSuffix = $portVersion
+	lib:libmbedcrypto$secondaryArchSuffix = $libVersion compat >= 6
```

Comment:
this resolves to "2.25.0 compat >= 6". It doesn't look correct to me.

Reply:
Wasn't sure about this, provides .so libs are 2.25.0 and 6

Reply:
Then "make" a correct version: "6.25.0 compat >= 6"
The point is to be able to make the differences when the major versions change (ie for 2.16.x it's 3, 12 and 0)
The alternative is to use the minor version: 2.25.0 compat >= 2.25, but one can be sure the compat won't break in 2.25.x

Reply:
Like in: libVersion="6.25.0" and set the compat to these lines?